CHANGELOG.md

```diff
@@ -1,3 +1,90 @@
+# v1.21.0 (2024-08-06)
+
+## OS Changes
+
+* Update host containers ([#4117])
+
+## Orchestrator Changes
+
+### Kubernetes
+
+* Enable k8s reserved cpus ([#3964])
+* Drop k8s 1.27 metal and VMware variants ([#4079])
+* Drop k8s 1.26 metal and VMware variants ([#4018]) 
+* Build the pause image from upstream and use as the default ([#3940]) thanks @tzneal!
+
+### ECS
+
+* Port to the ECS settings extension ([#3984])
+
+## Build Changes
+
+* Archive previous release migrations ([#4014])
+* Update Go dependencies ([#3999])
+
+### Twoliter
+
+* Remove leftover vendor section ([#4071])
+* Update Twoliter to 0.4.4 ([#4008], [#4086], [#4093], [#4123]) 
+* Update bottlerocket-core-kit to v2.3.1 ([#4122])
+* Migrate to core kit ([#4060])
```

Comment:
nit: Not sure if ordering matters, but in my head, it makes sense to reference the core-kit migration first before mentioning the updates to it